habitat/tasks/rearrange/grip_actions.py

```diff
@@ -18,7 +19,41 @@
 )
 
 
-class GripSimulatorTaskAction(SimulatorTaskAction):
+class RobotAction(SimulatorTaskAction):
+    _sim: RearrangeSim
+
+    @property
+    def _ik_helper(self):
```

Comment:
Both _ik_helper and cur_robot use the same logic : 
```
if self._config.AGENT is None:
            return self._sim.robots_mgr[0].XXX
        return self._sim.robots_mgr[self._config.AGENT].XXX
```
I would make a helper method : 
```
def _robot_mgr(self):
   if self._config.AGENT is None:
            return self._sim.robots_mgr[0]
   return self._sim.robots_mgr[self._config.AGENT]
```
And then have _ik_helper and cur_robot just call `self._robot_mgr().ik_helper` or `self._robot_mgr().robot`